src/olympia/bandwagon/models.py

```diff
@@ -61,8 +61,7 @@ class Collection(ModelBase):
     slug = models.CharField(max_length=30, blank=True, null=True)
 
     # description can (and sometimes does) contain html and other unsanitized
-    # content. It must be cleaned before display - NoURLsField just strips the
-    # URL without doing any escaping.
+    # content. It must be cleaned before display.
```

Comment:
We should reword that entire docstring now, as we are escaping, but at the same time it should be rendered as plain text by clients.